static/js/containers/ChannelPage.js

```diff
@@ -36,21 +49,30 @@ class ChannelPage extends React.Component {
   }
 
   renderContents(channelName, channels, postsForChannel, posts) {
+    if (!channels.data || !posts.data) {
+      return
+    }
     const channel = channels.data.get(channelName)
+    if (!postsForChannel.data) {
+      return
+    }
     const postIds = postsForChannel.data.get(channelName)
 
-    if (R.isNil(channel) || R.isNil(postIds)) {
+    if (!channel || !postIds || !posts.data) {
```

Comment:
This is here to appease flow which doesn't really understand `R.isNil`. When I made the types more specific flow had some warnings about these variables.